docs/mib-request.md

```diff
@@ -70,12 +70,18 @@ localMibs:
   pathToMibs: "/home/user/local_mibs"
 ```
 
-It creates a Kubernetes pvc with MIB files inside and maps it to MIB server pod.
-Also, you can change the storageClass and size of persistence according to the `mibserver` schema: [check here](https://github.com/pysnmp/mibs/blob/develop/charts/mibserver/values.yaml).
+To verify if the process of compilation was completed successfully, check the mibserver logs with:
 
-4. Whenever you add new MIB files, rollout restart MIB server pods to compile them again:
+```bash
+microk8s kubectl deployments/snmp-mibserver
```

Comment:
Shouldn't it be `microk8s kubectl logs deployments/snmp-mibserver` to get logs?

Reply:
Right, fixed!